app/Http/Controllers/Admin/OverviewController.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
please reformat and refactor this code. wierd spaced and tab usages.
lots and lots of code in the index, can be seperated in private methods or helpers for reusabillity and readability